docs/wiki/Home.md

```diff
@@ -12,28 +12,17 @@ The following instructions assume that you have already forked **dcrdex**.
 
 <img src="images/wiki-creation.png" width="700">
 
-2. In your terminal, navigate to your local **dcrdex** directory and clone the wiki from the **decred/dcrdex** repo.
+2. In your terminal, navigate to your local **dcrdex** directory and add your remote wiki.
 
-`git clone https://github.com/decred/dcrdex.wiki.git wiki`
+```
+git remote add wiki https://github.com/your-username/dcrdex.wiki.git
+git subtree push --prefix docs/wiki wiki master
```

Comment:
I'm assuming that wiki master does not yet exist here. I'm not sure tho... Will try to test will a different repo.

Reply:
Trying on a different repo, it looks like master is created with an initial commit. So, delete master and then push.